cognite/powerops/main.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import Annotated
+
+import typer
+
+app = typer.Typer()
+
+
+@app.command(
+    "plan",
+    help="Compare CDF resource creation between desired state (configuration files)"
+    " and remote state (resources in CDF) and print differences",
```

Comment:
Just a suggestion to the wording. Feel free to decline or rewrite again.
```suggestion
    help="Show the effect of applying the changes from the configuration files to the data model in CDF",
```